azure-storage-blob/tests/blob/test_container.py

```diff
@@ -404,8 +401,8 @@ def test_get_container_acl(self):
 
         # Assert
         self.assertIsNotNone(acl)
-        self.assertEqual(len(acl), 0)
-        self.assertIsNone(acl.public_access)
+        self.assertIsNone(acl.get('public_access'))
+        self.assertIsNone(acl.get('deserialized'))
```

Comment:
Should this be `acl.get('signed_identifiers')`?

Reply:
Yes, it's a typo, thanks for catching this. This is fixed in the second commit.